src/Symfony/Component/AssetMapper/Tests/ImportMap/RemotePackageStorageTest.php

```diff
@@ -46,18 +46,18 @@ public function testSaveThrowsWhenFailing()
         $vendorDir = self::$writableRoot.'/assets/acme/vendor';
         $this->filesystem->mkdir($vendorDir.'/module_specifier');
         $this->filesystem->touch($vendorDir.'/module_specifier/module_specifier.index.js');
-        $this->filesystem->chmod($vendorDir.'/module_specifier/module_specifier.index.js', 0555);
+        $this->filesystem->chmod($vendorDir.'/module_specifier', 0555);
 
         $storage = new RemotePackageStorage($vendorDir);
         $entry = ImportMapEntry::createRemote('foo', ImportMapType::JS, '/does/not/matter', '1.0.0', 'module_specifier', false);
 
         $this->expectException(\RuntimeException::class);
-        $this->expectExceptionMessage('file_put_contents('.$vendorDir.'/module_specifier/module_specifier.index.js): Failed to open stream: Permission denied');
+        $this->expectExceptionMessage('Failed to write file "'.$vendorDir.'/module_specifier/module_specifier.index.js');
 
         try {
             $storage->save($entry, 'any content');
         } finally {
-            $this->filesystem->chmod($vendorDir.'/module_specifier/module_specifier.index.js', 0777);
+            $this->filesystem->chmod($vendorDir.'/module_specifier/', 0777);
```

Comment:
🆘   The change from "chmod file" to "chmod folder" makes the tests fail on AppVeyor ..

This change is the only "solution" i found to create a permission problem for the test...
...  as filesystem has no problem writing in a file even with 0555...

If someone has an idea to solves this, that'd be very nice 🐱

Reply:
Any idea here maybe @derrabus ? 🥺 
(after my changes this test failed because no error was thrown anymore)

Reply:
on Windows, keeping an open file handle to the file might make replacing it fail